pkg/core/pipeline/autodiscovery/main.go

```diff
@@ -40,6 +42,7 @@ var (
 		"golang/gomod":  golang.Spec{},
 		"helm":          &helm.Spec{},
 		"helmfile":      &helmfile.Spec{},
+		"terraform":     &terraform.Spec{},
```

Comment:
I added this is while we were discussing i'm guessing it should now be removed due to platforms.

Reply:
This is one is needed to generate the jsonschema, and the one line 31 is useful as I can just run `updatecli diff --experimental` to see what would be changed.

I now often run `updatecli diff --experimental` when I look at a repository as it tells me how "outdated" the project is